src/lib/stitching/exchange/v2/stitching.ts

```diff
@@ -220,6 +222,53 @@ export const exchangeStitchingEnvironment = ({
       },
     },
   }
+  //  Experiment ----
+  const collectorProfileResolver = {
+    fragment: gql`
+      fragment CommerceOrderCollectorProfile on CommerceOrder {
+        buyer {
+          __typename
+          ... on User {
+            id
+          }
+        }
+      }
+    `,
+    resolve: async (parent, _args, context) => {
+      console.log("please help")
+
+      const { collectorProfileByUserIDLoader } = context
+      if (!parent.buyer.id || !collectorProfileByUserIDLoader) return null
+
+      const userId = parent.buyer.id
+
+      try {
+        const { body: profile } = await collectorProfileByUserIDLoader({
+          userId,
+        })
+        // loader returns an array
+        return profile[0]
+      } catch (e) {
+        console.log("error", e)
+        throw new GraphQLError(
+          `[metaphysics @ exchange/v2/stitching] CollectorProfile not found`
+        )
```

Comment:
Right now `collectorProfile` lives directly under `commerceOrder`. I would like to move it to be under `buyerDetails` section but I am a little afraid to touch it/having [trouble reading what is going on there](https://github.com/artsy/metaphysics/blob/main/src/lib/stitching/exchange/v2/stitching.ts#L46-L110). 

What do you think?